app_customers/lib/pages/dashboard/dashboard_view.dart

```diff
@@ -136,62 +140,132 @@ class _DashboardBody extends StatelessWidget {
+                      offset: Offset(0, 1),
+                      blurRadius: 12,
+                      spreadRadius: 0.5,
+                      color: AppColors.black,
+                    ),
+                  ],
+                ),
+                child: ClipRRect(
+                  borderRadius: BorderRadius.only(
+                    bottomLeft: Radius.circular(
+                      customDrawerController.raduis.toDouble(),
+                    ),
+                  ),
+                  child: TweenAnimationBuilder(
+                    tween: Tween<double>(
+                      begin: 0,
+                      end: customDrawerController.value.toDouble(),
+                    ),
+                    duration: const Duration(milliseconds: 500),
+                    builder: (_, val, __) {
+                      return AnimatedBottomNavigationBar.builder(
+                        itemCount: pages.length,
+                        tabBuilder: ((index, isActive) {
+                          final color = isActive
+                              ? theme.bottomNavigationBarTheme.selectedItemColor
+                              : theme
+                                  .bottomNavigationBarTheme.unselectedItemColor;
+
+                          return Column(
+                            mainAxisSize: MainAxisSize.min,
+                            mainAxisAlignment: MainAxisAlignment.center,
+                            children: [
+                              Icon(
+                                pages[index].iconData,
+                                size: 24,
+                                color: color,
+                              ),
+                              const SizedBox(height: 4),
+                              Padding(
+                                padding:
+                                    const EdgeInsets.symmetric(horizontal: 8),
+                                child: AutoSizeText(
+                                  pages[index].title,
+                                  maxLines: 1,
+                                  style: TextStyle(color: color),
+                                  group: autoSizeGroup,
+                                ),
+                              ),
+                            ],
+                          );
+                        }),
+                        height: 80,
+                        splashSpeedInMilliseconds: 300,
+                        backgroundColor: theme.bottomAppBarTheme.color,
+                        activeIndex: controller.activeIndex,
+                        gapLocation: GapLocation.end,
+                        notchSmoothness: NotchSmoothness.defaultEdge,
+                        shadow: BoxShadow(
+                          offset: Offset(0, 1),
+                          blurRadius: 12,
+                          spreadRadius: 0.5,
+                          color: AppColors.black,
+                        ),
+                        onTap: (int) {},
```

Comment:
Can you please clarify why the onTap action has been removed?

Reply:
ohh.. i must have delete it by mistake while resolving conflicts